assets/js/SignGroups.tsx

```diff
@@ -131,6 +172,23 @@ function SignGroupsForm({
 
   return (
     <form onSubmit={onSubmit}>
+      {changeGroupSignId && (
+        <ModalPrompt
+          acceptText="Yes, reassign this sign"
+          contents={changeGroupSignText(changeGroupSignId, line)}
+          elementId="tab-panel-container"
+          onAccept={() => {
+            const [station, zone] = changeGroupSignId;
```

Comment:
is there a reason that this is `[station, zone]` instead of being `signId`?

Reply:
Oh, you're right the naming is kind of inconsistent. But I should probably go the other way, and not call it `changeGroupSignId`.

Basically, I started with the idea that we were storing the sign ID in the state here (e.g. `orange_north_station_commuter_rail_exit`), hence the name of the hooks, and what is now `[station, zone]` used to be `signId` like you suggest. However, I ran into issues because there's not a great way to get from `orange_north_station_commuter_rail_exit` to `North Station CR Exit`. The station name and zone/platform description are encoded in our configuration indexed by the ARINC identifiers (see [here](https://github.com/mbta/signs_ui/blob/56795cedbc561e60bd4248cdaca0608506ce1da7/assets/js/mbta.ts#L283)). We also can go ARINC to internal ID [here](https://github.com/mbta/signs_ui/blob/56795cedbc561e60bd4248cdaca0608506ce1da7/assets/js/mbta.ts#L2922) (in order to handle Realtime Signs POSTs).

I considered sticking with the ID, which would entail inverting that second map, and then string splitting on the `-` to get the ARINC station and zone, and then use those to index into the configuration. Or build another view of that configuration from the internal IDs. But ultimately, I realized that the component that's actually pressing the button is dealing with the ARINC station and zone *already*, and it was the component turning it into the internal ID, so instead I just had that component pass the station and zone along directly.

Reply:
Thank you; I was looking in this function for the use of `station` and `zone` outside `signId`, but I see now that they're used separately in `changeGroupSignText`.